tsconfig.json

```diff
@@ -1,16 +1,36 @@
 {
   "compilerOptions": {
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "lib": ["es6"],
-    "module": "umd",
+    "target": "es5",
+    "module": "commonjs",
     "moduleResolution": "node",
-    "noImplicitAny": true,
-    "outDir": "./dist/",
+    "noEmit": true,
     "sourceMap": true,
-    "target": "es5"
+    "declaration": false,
+    "noImplicitAny": false,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "lib": [
+      "es2015",
+      "dom"
+    ],
+    "typeRoots" : [
+      "./node_modules/@types"
```

Comment:
Think it would be nice to add our typings to @types too. On our (backend services) SDK, we had months of explaining to do to a customer, who couldn't manage their definitions. (since this convention didn't exist back when we moved to TS)

Reply:
The only reason you would need to add to @types is if you are not the owner of a package and still would like to add type definitions for a package. I don't think this is needed.